lib/stdlib/src/dets_v9.erl

```diff
@@ -2635,9 +2635,8 @@ v_segment(H, SegNo, SegPos, SegSlot) ->
     BucketP = SegPos + (4 * ?SZOBJP * SegSlot),
     case catch read_bucket(H, BucketP, H#head.type) of
 	{'EXIT', Reason} -> 
-	    dets_utils:vformat("** dets: Corrupt or truncated dets file~n", 
-			       []), 
-	    io:format("~nERROR ~tp~n", [Reason]);
+            io:format("** dets: Corrupt or truncated dets file~nERROR ~tp~n",
+                      [Reason]);
```

Comment:
Should not this also be printed by logger ?

Reply:
From what I can till the v_segment function is debug functionality to start with and is never called in production, so I decided to leave it as is.

Reply:
Yes, this looks like a debug functionality, so it seems that io:format is fine.

Reply:
I just thought logging it on debug level would make more sense. But if it never happens in production I guess does not matter so much.